inc/networkport.class.php

```diff
@@ -1237,4 +1237,19 @@ static function getConnexityMassiveActionsSpecificities() {
       return $specificities;
    }
 
+   public function getName($options = []) {
```

Comment:
I think the new `computeFriendlyName` method should be used for this usage. Can you check if it is possible ?